packages/loader/container-loader/src/test/connectionStateHandler.spec.ts

```diff
@@ -146,41 +145,127 @@ describe("ConnectionStateHandler Tests", () => {
         deltaManagerForCatchingUp = new MockDeltaManagerForCatchingUp();
 
         connectionStateHandler = createHandler(
```

Comment:
Observation: The instances created here in `beforeEach` are not disposed.  Actually, none are disposed. So we're leaking a bunch of Timers, right?

And I think before calling dispose you've gotta disconnect or else dispose will assert.

Reply:
I think that does not matter as we never leave it in the state (at the end of test) with timer on (otherwise we would see Node process hanging, I think). But I'll add dispose to protect against future changes